content/docs/installation/upgrade.md

```diff
@@ -87,35 +87,4 @@ number you want to install:
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/<version>/cert-manager.yaml
 ```
 
-Once you have deployed the new version of cert-manager, you can [verify](verify.md) the installation.
-
-## Reinstalling cert-manager
-
-In some cases there may be a need to do a full uninstall and re-install of
-cert-manager. An example could be when a very old cert-manager version needs to
-be brought up to date and it isn't feasible to upgrade one minor version at a
-time, which is our default recommended upgrade strategy.
```

Comment:
I suggest leaving this paragraph here with a link to the new reinstall page.

Reply:
Ok